server/auth-router.js

```diff
@@ -101,9 +101,12 @@ module.exports = function authRouter(config = {}) {
 		info(`LoginUI: execute logout, session id:${req.sessionID}, cookie:${getSyncCookie(req)}, user id:${req.user && req.user.id}`); // eslint-disable-line max-len
 		const returnUrl = encodeURIComponent(`https://${config.host}`);
 		const logoutUrl = `https://${config.auth0.domain}/v2/logout?returnTo=${returnUrl}`;
-		req.logout(); // removes req.user
-		noteLoggedOut(res);
-		res.redirect(logoutUrl);
+		req.logout({}, err => {
```

Comment:
Looking at the example you linked in this PR, I think this should be:
`req.logout((err) => {` ?

Reply:
`err` is the only param passed into this callback.

The first param is an options object, but yea it may be ok to omit the empty `{}`.